plugin/ruby_hl_lvar.vim

```diff
@@ -14,31 +14,27 @@ let g:ruby_hl_lvar_show_warnings =
 
 augroup ruby_hl_lvar
 	autocmd!
-	autocmd! Filetype * call Ruby_hl_lvar_filetype()
+	autocmd Filetype ruby call Ruby_hl_lvar_filetype()
```

Comment:
I guess it does not work on `ruby.rspec` filetype.

Reply:
oops, Thanks.